cirq/linalg/__init__.py

```diff
@@ -53,6 +52,9 @@
     diagonalize_real_symmetric_matrix,
 )
 
+from cirq.linalg.distance_measures import (
```

Comment:
I think we're going to see two things happen. On the one hand, we should probably limit ourselves to a few most commonly used distance measures. On the other, we'll probably need other quantum information tools than distance measures, such as purity (see https://github.com/quantumlib/Cirq/issues/2763#issuecomment-585578953). Therefore, I think we need a somewhat more general name. Perhaps `cirq.qis` (for quantum information science) or `cirq.quantum_info` or `cirq.quantum_information`. FWIW, qiskit names their module "qiskit.quantum_info". @Strilanc do you have an opinion on this?

Also, I don't think this fits under `cirq.linalg` since this isn't linear algebra. Of course, under the hood it is implemented using linear algebra, but fidelity is certainly not a concept in linear algebra.

Reply:
After purity, another tool we're likely to see is some measure of entanglement. Having a QIS subpackage will allow us to prevent proliferation of subpackages each with a small number of publicly visible functions.

Reply:
Marking resolved since this is tracked by an issue and is almost done (PR open).